modules/core/shared/src/main/scala/gem/config/StaticConfig.scala

```diff
@@ -47,35 +48,38 @@ object StaticConfig {
 
   import GmosConfig._
 
-  final case class GmosNorth(
+  @Lenses final case class GmosNorth(
     common:    GmosCommonStaticConfig,
     stageMode: GmosNorthStageMode
   ) extends StaticConfig.Impl(Instrument.GmosN)
 
+  @SuppressWarnings(Array("org.wartremover.warts.PublicInference"))
   object GmosNorth extends GmosNorthLenses {
+
     val Default: GmosNorth =
       GmosNorth(
         GmosCommonStaticConfig.Default,
         GmosNorthStageMode.FollowXy
       )
+
   }
 
-  trait GmosNorthLenses {
-    val Common: GmosNorth @> GmosCommonStaticConfig =
-      Lens((a, b) => a.copy(common = b), _.common)
+  trait GmosNorthLenses { this: GmosNorth.type =>
+
+    lazy val customRois: Lens[GmosNorth, Set[GmosCustomRoiEntry]] =
+      common ^|-> GmosCommonStaticConfig.customRois
 
-    val CustomRois: GmosNorth @> Set[GmosCustomRoiEntry] =
-      Common >=> GmosCommonStaticConfig.CustomRois
+    lazy val nodAndShuffle: Lens[GmosNorth, Option[GmosNodAndShuffle]] =
```

Comment:
Perhaps we should add a `@>` type alias?

Reply:
I'd prefer to use the lib with its intended API for a while and see how it goes, but if it ends up being a pain then sure, we can add aliases. Same with `Either` and `\/` … I thought it would bug me but so far I don't care.